docs/Features.md

```diff
@@ -0,0 +1,25 @@
+# Features
+
+## XML Tag Auto Close
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides an `auto close` feature for XML tags:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag 1](images/Features/AutoCloseTag1.gif)
+
+There is also an autocomplete for adding an opening tag which adds the corresponding closing tag.
```

Comment:
Using autocomplete can be confusing because here you want to speak about auto close andnot completion. I would say more:

Using `>` at the end of an opening tag generates the corresponding closing tag.